ui/components/ui/url-icon/url-icon.js

```diff
@@ -3,6 +3,15 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import IconWithFallback from '../icon-with-fallback';
 
+/**
+ * @deprecated `<UrlIcon />` has been deprecated in favor of one of the avatar components `<AvatarNetwork />`, `<AvatarToken/>` or `<AvatarFavicon/>`
+ * component in ./ui/components/component-library/avatar-token/
+ * See storybook documentation for BannerAlert here:
+ * {@see {@link https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-avatartoken--docs}}
+ *
+ * Help to replace `UrlIcon` with `AvatarToken` by submitting a PR
+ */
+
```

Comment:
Adding deprecation message to old UrlIcon component

![Screenshot 2023-06-13 at 5 13 15 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/e8dda2ea-5762-4224-9f55-78824a9389bf)